Emergence/Contexts/Networking/XAppToken.swift

```diff
@@ -0,0 +1,64 @@
+import Foundation
+
+private extension NSDate {
+    var isInPast: Bool {
+        let now = NSDate()
+        return self.compare(now) == NSComparisonResult.OrderedAscending
+    }
+}
+
+struct XAppToken {
+    enum DefaultsKeys: String {
+        case TokenKey = "TokenKey"
+        case TokenExpiry = "TokenExpiry"
+    }
+
+    // MARK: - Initializers
+
+    let defaults: NSUserDefaults
+
+    init(defaults:NSUserDefaults) {
```

Comment:
I think you can merge both `init`s by writing a single one:

`init(defaults: NSUserDefaults = .standardUserDefaults)`